LFGBulletinBoard/GroupBulletinBoard.lua

```diff
@@ -341,99 +369,93 @@ end
 function GBB.BtnRefresh(button)
 	GBB.UpdateLfgTool()
 end
-
-
---Tag Lists
----------------------------------------------------
-
----Sets the `GBB.tagList` table, specified by locale. 
----@param loc string The locale to create the tag list for.
-function GBB.CreateTagListLOC(loc)
-	for _,tag in pairs(GBB.badTagsLoc[loc]) do
-		if GBB.DB.OnDebug and GBB.tagList[tag]~=nil then
-			print(GBB.MSGPREFIX.."DoubleTag:"..tag.." - "..GBB.tagList[tag].." / "..GBB.TAGBAD)
-		end	
+--------------------------------------------------------------------------------
+-- Tag Lists
+--------------------------------------------------------------------------------
+local tagCollisions
+---Sets the `GBB.tagList` table with the tags specified by the given locale.
+---@param locale string The locale to create the tag list for.
+local function setTagListByLocale(locale)
+	for _,tag in pairs(GBB.badTagsLoc[locale]) do
 		GBB.tagList[tag]=GBB.TAGBAD		
 	end
-	
-	for _, tag in pairs(GBB.searchTagsLoc[loc]) do
-		if GBB.DB.OnDebug and GBB.tagList[tag]~=nil then
-			print(GBB.MSGPREFIX.."DoubleTag:"..tag.." - "..GBB.tagList[tag].." / "..GBB.TAGSEARCH)
-		end
+	for _, tag in pairs(GBB.searchTagsLoc[locale]) do
 		GBB.tagList[tag]=GBB.TAGSEARCH		
 	end
-	
-	for _, tag in pairs(GBB.suffixTagsLoc[loc]) do
-		if GBB.DB.OnDebug and tContains(GBB.suffixTags,tag) then
-			print(GBB.MSGPREFIX.."DoubleSuffix:"..tag)
-		end	
```

Comment:
removed the debug warnings for collisions on these since the algorithm/addon should always prioritize keys `GBB.TAGBAD` and `GBB.TAGSEARCH` for any patterns set in the preset for those.